### ISSUE TITLE
Using the ctrl+space shorcut the "space" is input in the box

### ISSUE THREAD

**Issue by HenriAugusto** [HenriAugusto/completion-plugin#26]
You can set the plugin hotkey to something like "w" to better visualize it. The "w" will be input into the box.

**Comment by HenriAugusto** [comment]
Problem:

1. User press hotkey and instantly we go to
2. ::completion::trigger
3. which calls ::completion::popup_draw
4. which calls bind .pop.f.lb <KeyRelease> {::completion::lb_keyrelease %K %A}
5. pop up appears
6. user release keys
7. this calls ::completion::lb_keyrelease

By the way that problem happens since i've refactored the plugin. But since Tab (old hotkey) wasn't used by PD you wouldn't notice it

**Comment by HenriAugusto** [comment]
Also, we need a break

```TCL
bind "completion-plugin" <$completion::config(hotkey)> {
   ::completion::trigger;
   break;
}
```

**Comment by HenriAugusto** [comment]
Fixed in [a99b4c5](https://github.com/HenriAugusto/completion-plugin/commit/a99b4c576c49793ad2b0f66578d2b69ea628443b).
